modules/misc/news.nix

```diff
@@ -1325,6 +1325,13 @@ in
           A new module is available: 'programs.kitty'.
         '';
       }
+
+      {
```

Comment:
You can choose to leave out the news entry for new modules if you want. The merger will add it when merging. Then you won't suffer the merge conflict warnings.